.github/workflows/jest.yml

```
@@ -17,8 +17,7 @@ jobs:
     # Job steps
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-node@v4
-        with:
-          node-version: 20.x
+      - uses: actions/setup-node@v4       
+      - node-version: 23.1.0
```

actionlint (1.7.4)

unexpected key "node-version" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)